### ISSUE TITLE
fieldValueEquals fails on textarea with goutte driver

### ISSUE THREAD

**Issue by ifdattic** [minkphp/Mink#713]
Assuming you have HTML with textarea containing a value:

```html
<textarea id="comment" class="form-control" rows="3">{{ draft.comment }}</textarea>
```

And you run the following in the step:

```php
$this->assertSession()->fieldValueEquals('comment', $comment);
```

It would fail with Goutte driver as the value is null. This would work fine with Selenium driver. In my opinion it should work in both cases as you're not using any JS and the markup is correct.

Bellow is the dumps of the following lines:

```php
$commentField = $this->assertSession()->elementExists('css', '#comment');

var_dump($commentField->getText());
var_dump($commentField->getValue());
```

```bash
# with goutte driver
/home/vagrant/peerfeedback/features/bootstrap/StudentFeedbackContext.php:89:
string(31) "Very well done with a few typos"
/home/vagrant/peerfeedback/features/bootstrap/StudentFeedbackContext.php:90:
NULL

# with selenium driver
/home/vagrant/peerfeedback/features/bootstrap/StudentFeedbackContext.php:89:
string(31) "Very well done with a few typos"
/home/vagrant/peerfeedback/features/bootstrap/StudentFeedbackContext.php:90:
string(31) "Very well done with a few typos"
```

**Comment by aik099** [comment]
This seems to known limitation of Goutte driver because of underlying BrowserKit driver is treating `{{` like template placeholders. I'm not 100% sure, that this is the case here, but it might be.

**Comment by ifdattic** [comment]
@aik099 Forgot to change that one, it's actually the Twig variable so on the normal page it's just normal `Very well done with a few typos` text.

**Comment by aik099** [comment]
So this is isn't a problem anymore?

**Comment by ifdattic** [comment]
No, it's still the same issue. I just meant that it's actually normal text in HTML so `{{` are not there to be treated as template placeholders.

**Comment by aik099** [comment]
There is a test in driver test suite, that confirms, that Goutte can get current value from textarea (presuming that textarea initially had that value in HTML and it wasn't set later using JavaScript): https://github.com/minkphp/Mink/blob/master/driver-testsuite/tests/Form/GeneralTest.php#L168 (analog to `$commentField->getValue()` code from your example).

Maybe there is a broken HTML markup of page you're testing and being headless Goutte can't properly parse HTML and detect value inside textarea?

**Comment by ifdattic** [comment]
That scenario needs any JavaScript (that is how I found this as after removing the `@javascript` tag it started to fail).

There was a few markup issues, but after fixing them the issue was not fixed.

No, there is only a single textarea on that page.

**Comment by aik099** [comment]
Maybe you can send add failing test in https://github.com/minkphp/Mink/tree/master/driver-testsuite that will demonstrate the problem?
